rsmtool/test_utils.py

```diff
@@ -790,20 +790,26 @@ def check_report(html_file,
     report_errors = 0
     report_warnings = 0
 
+    # Setting raise_warnings to false if not in STRICT mode
+    if IGNORE_WARNINGS:
+        raise_warnings = False
+
     with open(html_file, 'r') as htmlf:
-        for line in htmlf:
+        for i, line in enumerate(htmlf):
```

Comment:
What is the enumerate for? I don't see `i` being used anywhere?

Reply:
@desilinguist Yeah, there were 70 errors or so, when not in STRICT mode and 90 in STRICT mode (With deprecation warnings). After this fix, running the tests normally does not throw any errors.

Reply:
Oh, I was trying out something but ended up not needing it. Should I revert this line?

Reply:
Yes, please go ahead.